README.md

```diff
@@ -1,205 +1,82 @@
-
-```bash
-curl -i -X POST 'http://127.0.0.1:7700/indexes/movies/documents' \
-  --header 'content-type: application/json' \
-  --data-binary @movies.json
-```
-
-### Search for Documents
-
-#### In command line
-
-The search engine is now aware of your documents and can serve those via an HTTP server.
-
-The [`jq` command-line tool](https://stedolan.github.io/jq/) can greatly help you read the server responses.
-
-```bash
-curl 'http://127.0.0.1:7700/indexes/movies/search?q=botman+robin&limit=2' | jq
-```
-
-```json
-{
-  "hits": [
-    {
-      "id": "415",
-      "title": "Batman & Robin",
-      "poster": "https://image.tmdb.org/t/p/w1280/79AYCcxw3kSKbhGpx1LiqaCAbwo.jpg",
-      "overview": "Along with crime-fighting partner Robin and new recruit Batgirl, Batman battles the dual threat of frosty genius Mr. Freeze and homicidal horticulturalist Poison Ivy. Freeze plans to put Gotham City on ice, while Ivy tries to drive a wedge between the dynamic duo.",
-      "release_date": 866768400
-    },
-    {
-      "id": "411736",
-      "title": "Batman: Return of the Caped Crusaders",
-      "poster": "https://image.tmdb.org/t/p/w1280/GW3IyMW5Xgl0cgCN8wu96IlNpD.jpg",
-      "overview": "Adam West and Burt Ward returns to their iconic roles of Batman and Robin. Featuring the voices of Adam West, Burt Ward, and Julie Newmar, the film sees the superheroes going up against classic villains like The Joker, The Riddler, The Penguin and Catwoman, both in Gotham City… and in space.",
-      "release_date": 1475888400
-    }
-  ],
-  "nbHits": 8,
-  "exhaustiveNbHits": false,
-  "query": "botman robin",
-  "limit": 2,
-  "offset": 0,
-  "processingTimeMs": 2
-}
-```
-
-#### Use the Web Interface
-
-We also deliver an **out-of-the-box [web interface](https://github.com/meilisearch/mini-dashboard)** in which you can test Meilisearch interactively.
-
-You can access the web interface in your web browser at the root of the server. The default URL is [http://127.0.0.1:7700](http://127.0.0.1:7700). All you need to do is open your web browser and enter Meilisearch’s address to visit it. This will lead you to a web page with a search bar that will allow you to search in the selected index.
 
-| [See the gif above](#demo)
+- **Search-as-you-type experience —** Provides relevant search results in less than 50 milliseconds.
+- **Typo tolerant** — Understands misspellings and adapts to return relevant results.
+- **Facetting and filtering** — Enhance your user's search experience with custom filters and build a faceted search interface in a few lines of code.
+- **Sort by** — Sort results based on price, date, or pretty much anything else your users need.
+- **Synonyms** — Meilisearch supports synonyms, so each of your users will find the content they are looking for.
+- **Geosearch** — Filter, sort documents around real-world geographic data.
+- **Language support** — Meilisearch supports all languages and detects them automatically.
```

Comment:
@ManyTheFish do you consider it's true? (I would say no)
Could you help us making it more "true"

Reply:
the truth is "**want to** support all languages" because the more I dig into linguistics, the more I discover language subtilities.
We may redirect to a page listing the languages we officially support, on the doc or on the Charabia repository. 🤔

Reply:
WDYT of "Meilisearch supports a wide variety of [languages](linkstodoc/charabia) and detects them automatically"

Reply:
@gui has suggested a change I merged, please check out the final version!